library/src/main/java/com/airbnb/android/airmapview/AirMapType.java

```diff
@@ -10,9 +10,9 @@
  */
 public class AirMapType {
 
-  private static final String ARG_MAP_DOMAIN = "map_domain";
-  private static final String ARG_FILE_NAME = "map_file_name";
-  private static final String ARG_MAP_URL = "map_url";
+  protected static final String ARG_MAP_DOMAIN = "map_domain";
+  protected static final String ARG_FILE_NAME = "map_file_name";
```

Comment:
Did you mean to make these `protected`? It doesn't look like they're used anywhere outside of this class